src/wms/src/main/java/org/geoserver/wms/map/RenderedImageMapOutputFormat.java

```diff
@@ -919,6 +922,25 @@ private RenderedImage directRasterRender(WMSMapContent mapContent, int layerInde
             tileSizeY = mapContent.getMapHeight();
         }
         
+        // 
+        // Band selection
+        //
+        
+        int[] bandIndicesTmp = null;
```

Comment:
This code is repeated in geotools as well. Could be factored out to a static method in the new style visitor for example.